tests/test_init.py

```diff
@@ -1,16 +1,15 @@
 from importlib import reload
+from importlib.metadata import PackageNotFoundError
 from unittest.mock import Mock, patch
 
 from django.test import SimpleTestCase
-from pkg_resources import DistributionNotFound
 
 
 class TestInit(SimpleTestCase):
     def test_package_is_not_installed(self):
         with patch(
-            "pkg_resources.get_distribution", Mock(side_effect=DistributionNotFound)
+            "importlib.metadata.version", Mock(side_effect=PackageNotFoundError)
```

Comment:
Thank you for making this patch, moving from legacy `pkg_resources` to `importlib`is a positive move forward to accessing modules metadata.

tldr; Looks good to me.

Reply:
Thanks. This really just reflects the changes that were made to `rest_framework_simplejwt/__init__.py` in https://github.com/jazzband/djangorestframework-simplejwt/commit/19cf38e4a440fd648e82a94a7998ac857d48a28d

Reply:
A few `tox` tests seems to be breaking, though. Mostly version-dependent tests.

Reply:
There are a few tests that seem to not have been passing for a while. I just opened #769 to consolidate the fixes, and I'll update this MR to remove the tests fixes.

Reply:
Oh great this should be ready for ready for merge then, @Andrew-Chen-Wang just merged #768